generator/test/error_test_data/nullable_realm_value.expected

```diff
@@ -0,0 +1,12 @@
+RealmValue fields cannot be nullable
+
+in: asset:pkg/test/error_test_data/nullable_realm_value.dart:7:3
+  ╷
+5 │ @RealmModel()
+6 │ class _Bad {
+  │       ━━━━ in realm model for 'Bad'
+7 │   RealmValue? wrong;
+  │   ^^^^^^^^^^^ RealmValue? is nullable
```

Comment:
In general I agree that we should have long descriptive error messages, but we need to agree on the team. Previously I was asked to simplify them.

Reply:
I would vote for short and clear messages. We could reword this to be more clear, but long error messages stand in the way after a while. So imo, this could be 
```
`RealmValue?` is declared nullable. Use `RealmValue` or `RealmValue myfield = RealmValue.nullValue()` to assign a default null value.
```

Reply:
I don't think it gets in the way. The short version is at the top, the rest you can read if you are confused.